classes/render_helpers/grading_report/sub_rows/multi_marker_feedback_sub_rows.php

```diff
@@ -44,22 +44,22 @@ class multi_marker_feedback_sub_rows implements sub_rows_interface {
     /**
      * @var bool
      */
-    protected $already_shown_a_new_buton = false;
+    protected $alreadyshownanewbuton = false;
```

Comment:
button?

Reply:
This var is not referred to elsewhere so removed in commit ee80f81